README.md

```diff
@@ -58,7 +58,7 @@ Alpha tasks:
   - [x] Add IP Device abstraction
   - [x] Add IP Device implementation based on go-tun2socks (LWIP)
   - [ ] Add UDP handler to fallback to DNS-over-TCP
-  - [ ] Add DelegatePacketProxy
+  - [x] Add DelegatePacketProxy for indirect UDP proxy
```

Comment:
It's not clear to me what that means.

Perhaps "to allow replacing the packet proxy at run time?"